apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatInfoView.kt

```diff
@@ -748,10 +752,40 @@ private fun MessageButton(openedFromChatView: Boolean, chat: Chat, contact: Cont
   )
 }
 
+@Composable
+fun SearchButton(chat: Chat, contact: Contact, close: () -> Unit) {
+  InfoViewActionButton(
+    icon = painterResource(MR.images.ic_search),
+    title = generalGetString(MR.strings.info_view_search_button),
+    disabled = !contact.ready || chat.chatItems.isEmpty(),
+    onClick = {
+      close.invoke()
+      withBGApi {
+        openDirectChat(chat.remoteHostId, contact.contactId, chatModel)
+        chatModel.chatViewMode.value = ChatViewMode.Search
+      }
+    }
+  )
+}
+
+@Composable
+fun MuteButton(chat: Chat, contact: Contact) {
+  val ntfsEnabled = remember { mutableStateOf(chat.chatInfo.ntfsEnabled) }
+
+  InfoViewActionButton(
+    icon =  if (ntfsEnabled.value) painterResource(MR.images.ic_notifications_off) else painterResource(MR.images.ic_notifications),
+    title = if (ntfsEnabled.value) stringResource(MR.strings.mute_chat) else stringResource(MR.strings.unmute_chat),
+    disabled = !contact.ready || !contact.active,
+    onClick = {
+      toggleNotifications(chat, !ntfsEnabled.value, chatModel, ntfsEnabled)
+    }
+  )
+}
+
 @Composable
 fun CallButton(chat: Chat, contact: Contact) {
   InfoViewActionButton(
-    icon = painterResource(MR.images.ic_call_filled),
+    icon = painterResource(MR.images.ic_call),
```

Comment:
it's a bit strange now that same disabled button looks differently in different cases - when member doesn't have contact it's filled (which does look better when inactive), when call is in progress it's similarly disabled but not filled

Reply:
one way could be to pass both variants to InfoViewButton.. or maybe always have it filled? depending on how we want it to look